bin/deploy.sh

```diff
@@ -107,6 +107,7 @@ aws s3 sync \
 
 # SVG; cache forever, assign correct content-type
 aws s3 sync \
+  --cache-control "max-age=${MAX_AGE}" \
```

Comment:
@ckprice It looks like `MAX_AGE` is a deceptively-named constant which only equals 600 seconds. To cache SVG forever, you want the bigger value found in the catch-all below (line 122), 31536000 seconds.

Here's what I'd suggest, since we're already in this file: 
- rename `MAX_AGE` to something more descriptive, like `TEN_MINUTES`, and update all existing instances
- add a new variable, maybe call it `ONE_YEAR`, with the value 31536000
- you'll also want to reassign `ONE_YEAR` to something dramatically shorter, like 60, in the "if dev" clause around line 45
- use `ONE_YEAR` in the line you added, and replace the number with the variable in the catch-all on line 122

This is a bit of a refactoring chore, but I probably would have made the same mistake, so it'll be worth the trouble. Ping me when it's ready for another look :+1: